_layouts/newsletter.html

```diff
@@ -3,18 +3,22 @@
 --- 
 <link rel="stylesheet" href="/assets/css/main.css">
 
-{% include functions/localization-finder.md %}
+{% assign baseurl = page.permalink | remove_first: "/" | remove_first: page.lang %}
 
 <article class="newsletter">
   <header class="post-header">
-    {% if localizations.size > 0 %}
-      <div class="localization">
-        <a href="{{ '/en' | append:baseurl }}">en</a>
-        {% for locale in localizations %}
-          | <a href="{{ locale.url }}">{{locale.lang}}</a>
-        {% endfor %}
-      </div>
-    {% endif %}
+    <div class="localization">
+      <a href="{{ '/en' | append:baseurl }}">en</a>
+      {% for lang in site.languages %}
```

Comment:
Is it possible to keep this code as a function in localization-finder.md rather than duplicate it on multiple layout templates?